documentation/adr/0011-version-project-with-cmake.md

```diff
@@ -0,0 +1,34 @@
+[<-previous](0010-package-dependencies-in-repo.md) | next->
+
+# 11 - Version project using CMake
+
+Date: 2020-Mar-25
+
+## Status
+
+Accepted
+
+## Context
+
+The project needs a versioning system. The version should be accessible from
```

Comment:
I'm not sure I understand what this means: "The project needs a versioning system."